apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/PDFGenerator.kt

```diff
@@ -139,7 +140,28 @@ class PDFGenerator(
 
         return brevbakerService
             .genererPdf(brev.id, brevRequest)
-            .also {
+            .let {
+                // TODO: finne et bedre sted for dette?
+                when (brev.brevkoder) {
+                    Brevkoder.OMS_INNTEKTSJUSTERING_VARSEL -> {
+                        val vedtaksbrev =
+                            brevbakerService.genererPdf(
+                                brev.id,
+                                BrevbakerRequest.fra(
+                                    EtterlatteBrevKode.OMSTILLINGSSTOENAD_REVURDERING, // TODO: gjennbruke eller egen ny mal for vedtaksbrev
+                                    OmstillingsstoenadInntektsjustering(), // TODO: må ha riktig data iht brevkode
+                                    avsender,
+                                    generellBrevData.personerISak.soekerOgEventuellVerge(),
+                                    sak.id,
+                                    generellBrevData.spraak,
+                                    sak.sakType,
+                                ),
+                            )
+                        PDFService.kombinerPdfListeTilEnPdf(listOf(vedtaksbrev, it))
+                    }
+                    else -> it
+                }
+            }.also {
```

Comment:
Litt smak og behag, men jeg synes kanskje ikke dette har noe å gjøre i generatoren. For meg burde en PDFgenerator kun ha ansvar for å generere noe basert på et datasett. Nå innføres det plutselig logikk som jeg tenker heller hører hjemme i en service.